docker-compose.yml

```diff
@@ -2,20 +2,21 @@ version: '3.1'
 
 services:
 
-  wordpress:
+  wordpress_dev:
```

Comment:
Can you explain the rename? Is it going to be disruptive to anyone's personal workflows?

Reply:
I wanted to better express the intent of this service and make it distinct from the image name. I can revert it now that I confirmrd that everything works

Reply:
Reverted in c0836cc. It isn't that important and I share your concern that it might cause some issues.